app/src/controllers/user.js

```diff
@@ -144,6 +159,19 @@ module.exports = {
     callback(client);
```

Comment:
Not sure. The idea was that a client would keep pinging for as long as they were working in a group -- as long as they kept the slot occupied, so to speak. Otherwise we can't distinguish between them leaving the slot vs choosing a circuit. Once they start playing the slot remains occupied until they choose to exit it, and then it becomes free.